Test/Case/Lib/DebugMemoryTest.php

```diff
@@ -46,18 +46,18 @@ public function testMemorySettingAndGetting() {
 		$this->assertTrue($result);
 
 		$result = DebugMemory::getAll(true);
-		$this->assertEquals(count($result), 1);
+		$this->assertCount(1, $result);
```

Comment:
The message isn't saying you added tabs - it's pointing out that the modified lines (or simply, the file) are tab indented. That is something which needs addressing.